lake/ztests/manage-compact.yaml

```
@@ -0,0 +1,20 @@
+script: |
+  export ZED_LAKE=test
+  zed init -q
+  zed create -S 10KB -q test
+  zed use -q test
+  for i in {1..10}; do
+  seq 200 | zq '{ts:this}' - | zed load -q -
+  done
+  zed manage compact -q -config manage.conf
+  zed query -z 'from test@main:objects | yield meta'
+
+inputs:
+  - name: manage.conf
```

I think manage.yaml should be the conventional name for this.
```suggestion
  - name: manage.yaml
```